models/pardot/base.sql

```diff
@@ -1,8 +1,8 @@
-drop schema if exists pardot cascade;
-create schema pardot;
+drop schema if exists pardot_analysis cascade;
```

Comment:
this seems scary to me... `pardot_analysis` is definitely more uniquely identifying than `pardot`... can't help but think that maybe we should prefix these schemas. Something like `ac_pardot` or `__ac_pardot` ...? That would make cleaning up schemas (as mentioned below) easy as well